### ISSUE TITLE
&РегламентноеЗадание/@Scheduled

### ISSUE THREAD

**Issue by nixel2007** [autumn-library/autumn#93]
Асинх-таска, которая выполняется с расписанием. 

Предположительно нужно создать рогатку (отключаемую), которая будет в цикле греть процессор и запукать задачи. таймауты для приостановить можно попробовать вычислять по данными зарегистрированных задач, но кажется, там чуть ли не линейные уравнения надо будет решать в таком случае. хотя какой-то примитив типа "не реже чем раз в 15 секунд или с периодичностью самой частой задачи с таймаутом меньше 15 секунд" можно сделать.

**Comment by Nivanchenko** [comment]
А может не греть процессор, а таймаут ставить в размере до следующего запускат рз? 
Ну типо РЗ каждый день в 9:00. Тогда фз стартует и делает Приостановить(количествоМилисекДо900Завтра) ?

**Comment by nixel2007** [comment]
Когда рз одно - это ок. А когда их 10?

**Comment by nixel2007** [comment]
Можно конечно стартовать 10 фоновых с приостановить...

**Comment by Nivanchenko** [comment]
> Можно конечно стартовать 10 фоновых с приостановить...

Согласен, тут из двух зол выбираем. Либо крутить циклы, либо несколько ФЗ спящих.
Кмк, если у нас их 10, и все долгие, а одно 15 сек. То каждые 15 сек гнать цикл и проверять все 10 на то, что нужно ли их запускать избыточно.
